src/classes/ERR_Handler.cls

```diff
@@ -449,10 +907,10 @@ public class ERR_Handler {
     */
     private static String getStatusCode(system.StatusCode status) {
         if (status == system.Statuscode.REQUIRED_FIELD_MISSING)
-            return 'Required Field Missing';   
+            return 'Required Field Missing';
         else if (status == system.Statuscode.FIELD_CUSTOM_VALIDATION_EXCEPTION)
             return 'Custom Validation Rule Failed';
         else
-            return 'Other';
+            return STATUS_CODE_OTHER_VAL;
```

Comment:
Curly brackets needed

Reply:
Another method that was here originally...I just refactored to use constants...I debated converting to Custom Labels but thought I would see what you thought...